### PR TITLE
fix email onboarding

### PR DESCRIPTION
**Pull request by Tschuck**
- [CORE-1051]

[CORE-1051]: https://evannetwork.atlassian.net/browse/CORE-1051

### CODE REVIEW

File: dapps/assets/src/components/contacts/InviteDispatcher.ts [hunk 1]
```diff
@@ -105,7 +105,7 @@ dispatcher
           fromAlias: data.fromAlias,
           lang: data.currLang,
         },
-        runtime.web3.utils.toWei('1'),
+        runtime.web3.utils.toWei('0'),
```
Comment (datrinh) [inline dapps/assets/src/components/contacts/InviteDispatcher.ts]: I think this caused an error in the backend the last time I tried to send 0. Did anything change there?

File: dapps/onboarding.vue/src/components/accept-contact/accept-contact.ts [hunk 2]
```diff
@@ -133,13 +133,15 @@ export default class AcceptContact extends mixins(EvanComponent) {
         accountId: this.runtime.activeIdentity,
       });
 
-
       // load my address book
       await this.runtime.profile.loadForAccount(this.accountId,
         this.runtime.profile.treeLabels.addressBook);
       // search for the target public key
-      const targetProfile = await bccHelper.getProfileForAccount(bcc,
```
Comment (datrinh) [inline dapps/onboarding.vue/src/components/accept-contact/accept-contact.ts]: Any reason why this helper function has to be replaced?

File: dapps/onboarding.vue/src/components/accept-contact/accept-contact.vue [hunk 3]
```diff
@@ -42,47 +47,58 @@
             </h5>
           </template>
           <template v-slot:footer>
-            <button type="button" class="btn  btn-primary font-weight-normal"
-              @click="$refs.logoutModal.hideModal()">
+            <button
+              type="button"
+              class="btn  btn-primary font-weight-normal"
```
Comment (datrinh) [inline dapps/onboarding.vue/src/components/accept-contact/accept-contact.vue]:
```suggestion
              class="btn btn-primary font-weight-normal"
```

File: dapps/onboarding.vue/src/components/accept-contact/accept-contact.vue [hunk 4]
```diff
@@ -42,47 +47,58 @@
...
         </evan-modal>
 
         <div class="mt-3">
-          <evan-success v-if="accepted"></evan-success>
-          <div class="text-center m-3" v-if="!accepted">
-            <evan-loading v-if="accepting"></evan-loading>
-            <button type="submit" class="btn btn-primary btn-block"
+          <evan-success v-if="accepted" />
+          <div
+            v-if="!accepted"
```
Comment (datrinh) [inline dapps/onboarding.vue/src/components/accept-contact/accept-contact.vue]:
```suggestion
            v-else
```

File: dapps/onboarding.vue/src/components/accept-contact/accept-contact.ts (hunk 2 above)
Comment (Tschuck) [reply dapps/onboarding.vue/src/components/accept-contact/accept-contact.ts]: This was the actual bug of this pr. This function does not exist anymore :D

File: dapps/assets/src/components/contacts/InviteDispatcher.ts (hunk 1 above)
Comment (Tschuck) [reply dapps/assets/src/components/contacts/InviteDispatcher.ts]: Yes. The backend can now handle the sending of zero eve :)

File: dapps/onboarding.vue/src/components/accept-contact/accept-contact.ts (hunk 2 above)
Comment (datrinh) [reply dapps/onboarding.vue/src/components/accept-contact/accept-contact.ts]: Haha, great :D

**Pull request merged**